### ISSUE TITLE
[Bug] Js and Css are injected twice on the text.

### ISSUE THREAD

**Issue by Palloxin** [LNReader/lnreader#664]
### Steps to reproduce

1. Take a freshly installed version of lnreader 1.1.19
2. AllNovelFull, Hidden marriage, add to the library
3. Download chapter 1 and 2
4. Inject this JS:
 ```js
alert(chapterName)
```
5. Open chapter 1, scroll until the bottom and click "Next chapter" (see video below).

### Expected behavior

- Js and css applied only once so the alert should popup only once.

### Actual behavior

Js applied twice when the chapters in question are downloaded.

https://github.com/LNReader/lnreader/assets/75091899/023cfea4-c12b-4542-b595-de249ea78afa

### LNReader version

1.1.18 and 1.1.19(the bug exists since the first time i started using lnreader so.. 1.1.15.1, idk previous versions)

### Android version

Android 8 Miui 8

### Device

huawei p8 lite 2017

### Other details

This issue has been almost completely rewritten on ``4th November 2023`` 
It is a re-opened issue because devs couldn't reproduce it even with the video so i think one needs to guess the cause of it correctly to fix it or test it on android 8.

EDIT: another re-edit in 9th of June 2024

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.15](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Palloxin** [comment]
I installed https://github.com/LNReader/lnreader/actions/runs/5435409646 and it doesn't happen anymore

https://github.com/LNReader/lnreader/assets/75091899/53917316-451f-4018-9e13-2979bd52ca72

**Comment by Palloxin** [comment]
Nvm, it just happens less with that. But with that fix it doesn't happen 100% of the times anymore

**Comment by rajarsheechatterjee** [comment]
Not able to reproduce this with simpler scripts. 

Your custom JS might be replacing / setting the HTML content more than once which could probably cause this.

**Comment by Palloxin** [comment]
iirc it has been fixed by danisty in the 1.1.17

**Comment by Palloxin** [comment]
https://github.com/LNReader/lnreader/assets/75091899/c6cf7fb1-5c7d-491b-90f6-abf5792f9afa

Here you can see that the JS has been applied already when the chapter loaded, then It lagged a bit, the chapter resetted and the JS applied again before the chapter_progress_resume feature activated. You can see it by
